server/sql/sell-everywhere-script.sql

```diff
@@ -0,0 +1,353 @@
+(
+    id uuid DEFAULT uuid_generate_v4(),
+    by_default BOOLEAN DEFAULT FALSE,
+    full_name VARCHAR(60),
+    country VARCHAR(40),
+    line1 VARCHAR(100),
+    line2 VARCHAR(100),
+    territory_type VARCHAR(70),
+    city VARCHAR(50),
+    post_code VARCHAR(10),
+    phone VARCHAR(20),
+    email_address VARCHAR(50),
+    customer_id uuid NOT NULL,
+    created_date TIMESTAMP DEFAULT NOW(),
+    updated_date TIMESTAMP DEFAULT NOW(),
+    CONSTRAINT FK_shipping_info_customer
+    FOREIGN KEY (customer_id) REFERENCES Customer(id) ON DELETE CASCADE,
+    unique(id)
+);
+
+drop table Shipping_Info_Customer;
+
+
+CREATE TABLE IF NOT EXISTS shipping_info
+(
+    id VARCHAR(36) PRIMARY KEY NOT NULL,
+    by_default BOOLEAN DEFAULT FALSE,
+    full_name VARCHAR(60),
+    country VARCHAR(40),
+    line1 VARCHAR(100),
+    line2 VARCHAR(100),
+    territory_type VARCHAR(70),
+    city VARCHAR(50),
+    post_code VARCHAR(10),
+    phone VARCHAR(20),
+    email_address VARCHAR(50),
+    checkout_id uuid NOT NULL,
+    created_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+       CONSTRAINT FK_shipping_info_checkout
+     FOREIGN KEY (checkout_id) REFERENCES Checkout(id) ON DELETE CASCADE
+);
+
+drop table Shipping_Info;
+
+CREATE TABLE IF NOT EXISTS Shipping_Info_Customer
+(
+    id uuid DEFAULT uuid_generate_v4(),
+    by_default BOOLEAN DEFAULT FALSE,
+    full_name VARCHAR(60),
+    country VARCHAR(40),
+    line1 VARCHAR(100),
+    line2 VARCHAR(100),
+    territoryType VARCHAR(70),
+    city VARCHAR(50),
+    post_code VARCHAR(10),
+    phone VARCHAR(20),
+    email_address VARCHAR(50),
+    checkout_id uuid NOT NULL,
+    created_date TIMESTAMP DEFAULT NOW(),
+    updated_date TIMESTAMP DEFAULT NOW(),
+       CONSTRAINT FK_shipping_info_checkout
+     FOREIGN KEY (checkout_id) REFERENCES Checkout(id) ON DELETE CASCADE,
+unique(id)
+);
+
+
+DROP TABLE Shipping_Info_Customer ;
+
+
+--IMAGE
+CREATE TABLE IF NOT EXISTS Image
+(
+    id uuid DEFAULT uuid_generate_v4() primary key,
+    name VARCHAR(100) NOT NULL,
+    endpoint VARCHAR(100) DEFAULT 'http://localhost:3001/uploads/',
+    created_date TIMESTAMP DEFAULT NOW(),
+    updated_date TIMESTAMP DEFAULT NOW()
+);
+
+ALTER TABLE image
+  ADD COLUMN primary_image BOOLEAN DEFAULT FALSE;
+
+ALTER TABLE Image
+ADD COLUMN endpoint  VARCHAR(100) DEFAULT 'http://localhost:3001/uploads/';
+
+drop table Image;
+
+
+CREATE TABLE IF NOT EXISTS stock_image
+(
+    id uuid DEFAULT uuid_generate_v4() primary key,
+    order_num INTEGER NOT NULL,
+    stock_id uuid,
+    image_id uuid,
+    CONSTRAINT FK_product_image_product
+     FOREIGN KEY (stock_id) REFERENCES Stock(id) ON DELETE CASCADE,
+    CONSTRAINT FK_product_image_image
+     FOREIGN KEY (image_id) REFERENCES Image(id) ON DELETE cascade,
+    created_date TIMESTAMP DEFAULT NOW(),
+    updated_date TIMESTAMP DEFAULT NOW()
+);
+
+ALTER TABLE image
+ ADD COLUMN "order" numeric not null;
+--IMAGE END
```

Comment:
check for holistic.dev
https://app.holistic.dev/en/shared/analytics/check-result/d4e9f5899a4c8221b698fe9496be1a21440e7dab2

it will help to improve

Reply:
new holistic.dev check results https://app.holistic.dev/en/shared/analytics/check-result/d6eb98800d393a311e46af383ace37b5f00a9ec0d

Reply:
@nosovk new holistic.dev results: https://app.holistic.dev/en/shared/analytics/check-result/d3c2402e0060f478833dc907fba3dfb3ad7fca6cb